app/src/main/res/layout/fragment_settings.xml

```diff
@@ -1,250 +1,405 @@
-        android:layout_height="wrap_content"
-        android:layout_marginStart="20dp"
-        android:layout_marginTop="40dp"
-        android:fontFamily="@font/lato_bold"
-        android:text="@string/user_name"
-        android:textColor="@color/grey"
-        android:textSize="17sp"
-        app:layout_constraintBottom_toTopOf="@id/userGroup"
-        app:layout_constraintStart_toEndOf="@id/userAvatar"
-        app:layout_constraintTop_toTopOf="@id/rectangleProfile" />
-
-    <TextView
-        android:id="@+id/userGroup"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginStart="20dp"
-        android:layout_marginBottom="40dp"
-        android:fontFamily="@font/lato_bold"
-        android:text="@string/user_group"
-        android:textColor="@color/grey"
-        android:textSize="11sp"
-        app:layout_constraintBottom_toBottomOf="@id/rectangleProfile"
-        app:layout_constraintStart_toEndOf="@id/userAvatar"
-        app:layout_constraintTop_toBottomOf="@id/userName" />
-
-    <ImageButton
-        android:id="@+id/infoProfile"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginEnd="10dp"
-        android:padding="10dp"
-        android:adjustViewBounds="true"
-        android:background="#00ffffff"
-        android:src="@drawable/info_profile_icon"
-        app:layout_constraintBottom_toBottomOf="@id/rectangleProfile"
-        app:layout_constraintEnd_toEndOf="@id/rectangleProfile"
-        app:layout_constraintTop_toTopOf="@id/rectangleProfile"
-        android:contentDescription="@string/todo" />
-
-    <androidx.constraintlayout.widget.Group
-        android:layout_width="0dp"
-        android:layout_height="0dp"
-        android:visibility="visible"
-        app:constraint_referenced_ids="rectangleTheme, themeAuto, themeLight, themeDark" />
-
-    <ImageView
-        android:id="@+id/rectangleTheme"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        app:layout_constraintTop_toBottomOf="@id/rectangleProfile"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        android:layout_marginTop="10dp"
-        android:src="@drawable/rectangle_theme"
-        android:contentDescription="@string/todo" />
-
-    <ImageButton
-        android:id="@+id/themeAuto"
-        android:layout_width="109dp"
-        android:layout_height="48dp"
-        android:src="@drawable/theme_defoult_auto_icon"
-        android:backgroundTint="#00ffffff"
-        app:layout_constraintStart_toStartOf="@id/rectangleTheme"
-        app:layout_constraintTop_toTopOf="@id/rectangleTheme"
-        app:layout_constraintBottom_toBottomOf="@id/rectangleTheme"
-        app:layout_constraintEnd_toStartOf="@id/themeLight"
-        android:contentDescription="@string/todo" />
-
-    <ImageButton
```

Comment:
We should not use the final values of the button size parameters. Need fix it in "@+id/themeLight" "@+id/themeAuto" android:id="@+id/userAvatar". But we can try to merge the branches, and then see how it will look and, if anything, fix it.